xcp_d/workflow/cifti.py

```diff
@@ -91,13 +92,16 @@ def init_ciftipostprocess_wf(
                 despike=False,
                 dcan_qc=False,
                 n_runs=1,
+                input_type,
                 layout=None,
+                input_type,
```

Comment:
`input_type` is in here twice. Also, without feeding it in as a keyword argument (e.g., `input_type="fmriprep"`), it would fail, since it comes after keyword arguments (e.g., `n_runs=1`).